### PR TITLE
Change notice styles on mobile.

### PR DESCRIPTION
**Pull request by danielgent**
## Summary

Addresses issue #2886

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/sass/components/user-input/googlesitekit-user-input-module.scss [hunk 1]
```diff
@@ -60,7 +60,7 @@
 
 	.googlesitekit-user-input__note {
 		color: $c-user-input-note;
-		margin: 0 0 0 43px;
+		margin: 0;
```
Comment (danielgent) [inline assets/sass/components/user-input/googlesitekit-user-input-module.scss:63]: screen5-after

File: assets/sass/components/user-input/googlesitekit-user-input-module.scss [hunk 2]
```diff
@@ -60,7 +60,7 @@
 
 	.googlesitekit-user-input__note {
 		color: $c-user-input-note;
-		margin: 0 0 0 43px;
```
Comment (danielgent) [inline assets/sass/components/user-input/googlesitekit-user-input-module.scss:63]: question5-before

File: assets/sass/components/user-input/googlesitekit-user-input-preview.scss [hunk 3]
```diff
@@ -33,8 +33,7 @@
 			background-color: $c-alabaster;
 			color: $c-cape-cod;
 			display: block;
-			margin-left: auto;
-			width: 85%;
+			width: 100%;
```
Comment (danielgent) [inline assets/sass/components/user-input/googlesitekit-user-input-preview.scss:36]: preview-after

File: assets/sass/components/user-input/googlesitekit-user-input-preview.scss [hunk 4]
```diff
@@ -33,8 +33,7 @@
 			background-color: $c-alabaster;
 			color: $c-cape-cod;
 			display: block;
-			margin-left: auto;
```
Comment (danielgent) [inline assets/sass/components/user-input/googlesitekit-user-input-preview.scss:36]: preview-before

**Review by eugene-manuilov — APPROVED** [review]
Awesome! Looks good to me.

**Pull request merged**